### ISSUE TITLE
Why my posts can delete other users?

### ISSUE THREAD

**Issue by dulebamboocha** [mhmoudalkahlout/laravel-blog#1]
Hi, mhmoudalkahlout,
I found your project in comments on Youtube tutorial of Alex J. Curtis 'How to Build a Blog with Laravel 5 Series''. You did awesome job. But I have question for you, same question I sended to Alex but no answer yet.
I almost went through all Alex's 'How to Build a Blog with Laravel 5 Series' and I figured out that when some user create post then other users can delete and edit his post. In other words said, all posts of some user appear to other users in their account and there they can delete and edit that posts which means that anyone can delete anyone's post. And I think it should not work that way. Is there some solution for that? If you have time to find solution can you update your project with that solution? This project is awesome but with ability that each user can see, on his own account, just his own posts, and that just he can modify and delete own posts, I think that project will be amazing!
Thanks for help!!!

**Comment by mhmoudalkahlout** [comment]
yes you are right unfortunately and you have to write your own code to achieve your idea.. as a help i can tell you to to add an extra filed in posts table (user_id) to store a logged in user id.. and you have to use session to store user id after login and when (delete, edit or view as you wish) check if the id in the session is same in the post (user_id) field >> if it is incorrect condition then deny access the action
